src/services/analytics/types.ts

```diff
@@ -40,7 +47,12 @@ export interface SwapSuccessEvent {
 export interface TransactionErrorEvent {
   error: string;
   errorCode?: string;
-  transactionType?: "payment" | "pathPayment" | "swap" | "sorobanToken";
+  transactionType?:
+    | "payment"
+    | "pathPayment"
+    | "swap"
+    | "sorobanToken"
+    | "sendCollectible";
```

Comment:
should we move the transactionType type to a shared type since it's being used on both TransactionSuccessEvent and TransactionErrorEvent?